static/src/javascripts/es6/projects/common/modules/crosswords/main.js

```diff
@@ -368,6 +368,23 @@ var Crossword = React.createClass({
         let focussed = this.clueInFocus(),
             isHighlighted = (x, y) => focussed ? helpers.entryHasCell(focussed, x, y) : false;
 
+        // Deep equal version of _.intersection
+        let findIntersectingCells = (array1, array2) =>
+            array1.filter(cell1 => array2.some(cell2 => _.isEqual(cell1, cell2)));
+
+        let focussedCells = focussed ? helpers.cellsForEntry(focussed) : [];
+        let entryHasIntersectingCell = entry => {
+            let cells = helpers.cellsForEntry(entry);
+            let intersecting = findIntersectingCells(cells, focussedCells);
+            return !! intersecting.length;
+        };
+
+        let otherEntries = _.difference(this.props.data.entries, focussed ? [focussed] : []);
+        let intersectingEntries = otherEntries.filter(entryHasIntersectingCell);
+
+        let cellIntersectsFocussedEntry = (x, y) =>
+            intersectingEntries.some(entry => helpers.entryHasCell(entry, x, y));
+
```

Comment:
Computationally this is a bit expensive.

If you have n clues, of average m length, you'll be doing nm² operations to get the intersecting clues.

So for this crossword, for example - http://m.code.dev-theguardian.com/crosswords/quick/13864 - you'd be doing 26 * 6 * 6 (936) passes each time.

I doubt that's going to cause any issues for any of the devices accessing the crosswords, but if we do see problems, we could look at caching the intersections on start up, as it's something you can just pre-compute. But don't do that unless we actually see it as a problem at some point, no point in prematurely optimising!